assets/js/covid19.js

```diff
@@ -106,6 +106,18 @@
             }
             return false;
         });
+
+        var browserLanguage = navigator.language.slice(0, 2);
```

Comment:
In ally-ui's `TranslateUtil` we have:
```
const langs = navigator.languages;
            if (langs.length > 0) {
                return navigator.languages[0];
            } else if (NullUtil.hasValue(navigator.language)) {
                return navigator.language;
            } else {
                return null;
            }
```

I can't recall the semantics atm, but can you check whether we need to have a fallback like that here?

Reply:
Frigging IE11. I add a similar fallback that should catch all options.